### PR TITLE
streamline parents lookup per run.

### PR DESCRIPTION
**Pull request by sephirothkod**
remove database column for parents and just use relationships look up each time.

### CODE REVIEW

File: app/models/bulkrax/importer_run.rb [hunk 1]
```diff
@@ -5,6 +5,8 @@ class ImporterRun < ApplicationRecord
     belongs_to :importer
     has_many :statuses, as: :runnable, dependent: :destroy
 
-    serialize :parents, Array
+    def parents
```
Comment (alishaevn) [inline app/models/bulkrax/importer_run.rb:8]: does this get called now with `importer.last_run.parents` in the schedule relationships job?

Comment (alishaevn) [inline app/models/bulkrax/importer_run.rb:8]: does this get called now with `importer.last_run.parents` in the schedule relationships job?

Comment (sephirothkod) [reply app/models/bulkrax/importer_run.rb:8]: yes; that is the idea at least.

**Review by alishaevn — APPROVED** [review]
this is working for me locally in louisville

**Pull request merged**